### ISSUE TITLE
Document Accessibility Issue: NVDA Fails to Announce the New Channel Logo Issue #5298.

### ISSUE THREAD

**Issue by Jay2006sawant** [WikiEducationFoundation/WikiEduDashboard#6091]
With reference to  #5298 
### What is happening?

While using NVDA, the new channel logo is not announced. This creates accessibility barriers for blind users who rely on screen readers to navigate the dashboard

-----------------------------------------------------------------------------------------------------------------------

### To Reproduce

-  Open the Wiki Education Dashboard.

- Use NVDA to navigate to the new channel logo.

- Move the cursor over the new channel logo.
 
- Observe that NVDA does not announce the logo.
------------------------------------------------------------------------------------------------------------------------
### Expected behavior
When using NVDA, the new channel logo should be announced when the cursor hovers over it. This will ensure that blind users who rely on screen readers can navigate the dashboard effectively.

------------------------------------------------------------------------------------------------------------------------
### Screenshots
![Screenshot 2025-01-06 033552](https://github.com/user-attachments/assets/2badf3a8-48ad-4f9a-8771-e1deb167ca28)

### Additional context
To install NVDA on Windows:

- Download NVDA from [official website ](https://www.nvaccess.org/download/)
- Run the installer and follow the on-screen instructions to complete the installation.
- Once installed, you can start NVDA by pressing Ctrl + Alt + N or by searching for "NVDA" in the Start Menu

**Comment by Jay2006sawant** [comment]
I am currently working on this issue and will be raising a  PR soon

**Comment by ragesoss** [comment]
What is the 'new channel logo'?

**Comment by Jay2006sawant** [comment]
@ragesoss Thank you for addressing the typo mistake. I have made necessary changes  from 'new' to 'news' in the issue mentioned above . I apologize for this mistake.

**Comment by ragesoss** [comment]
Ah! Do you mean the news icon next to the 'Log out' link?

**Comment by Jay2006sawant** [comment]
Yes
